src/frontend/src/tables/Details.tsx

```diff
@@ -312,14 +320,16 @@ function TableAnchorValue(props: FieldProps) {
     }
   });
 
+  const detailUrl = useMemo(() => {
+    return getDetailUrl(props.field_data.model, props.field_value);
+  }, [props.field_data.model, props.field_value]);
+
   return (
     <Suspense fallback={<Skeleton width={200} height={20} radius="xl" />}>
       <Anchor
-        href={
-          '/platform' + data.url ?? props.field_data.dest + props.field_value
-        }
-        target={data.external ? '_blank' : undefined}
-        rel={data.external ? 'noreferrer noopener' : undefined}
+        href={`/platform${detailUrl}`}
```

Comment:
@SchrodingersGat this is hard coded for the default PUI extension (that might not stay this way) - should we maybe make this respect the settings?